PlayCover/Utils/ImageCache.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  ImageCache.swift
+//  PlayCover
+//
+//  Created by Isaac Marovitz on 17/10/2022.
+//
+
+import Foundation
+
+class ImageCache {
```

Comment:
There isn't much of a reason to implement our own image cache when implementations like https://github.com/lorenzofiamingo/swiftui-cached-async-image already exist.